feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -214,4 +215,49 @@ export class ConfiguratorAttributeBaseComponent {
       uiType === Configurator.UiType.DROPDOWN_ADDITIONAL_INPUT
     );
   }
+
+  protected isRequiredErrorMsg(attribute: Configurator.Attribute): boolean {
+    if (attribute.required && attribute.incomplete) {
```

Comment:
Replace following lines:
```suggestion
   return attribute.required && attribute.incomplete || false;
```

Reply:
Done!